src/ui/dashboardpage.h

```diff
@@ -1,8 +1,8 @@
-#ifndef DASHBOARDPAGE_H
-#define DASHBOARDPAGE_H
+#pragma once
```

Comment:
Please do not use `#pragma once`. This is not portable and causes often errors in larger projects.

Reply:
<img width="498" alt="image" src="https://github.com/fair-acc/opendigitizer/assets/23313867/1bff236e-db83-43b0-b6d7-1f8a9ff06d4c">

Reply:
please don't.